odoo/fields.py

```diff
@@ -1979,27 +1980,30 @@ def _convert(self, value, record, validate):
             'strip_classes': self.strip_classes
         }
 
+        # TODO: Wouldn't it be better to move all the normalize stuff here
+        #       instead of having it called by the sanitizer? (html_sanitize)
+        #       Feels weird that only sanitized fields are normalized.
+
         if self.sanitize_overridable:
             if record.user_has_groups('base.group_sanitize_override'):
                 return value
 
-            original_value = record[self.name]
+            original_value = html_normalize_post(html_normalize_pre(record[self.name]))
             if original_value:
-                initial_value_sanitized = html_sanitize(original_value, **sanitize_vals)
+                original_value_sanitized = html_sanitize(original_value, **sanitize_vals)
```

Comment:
so in this case, you will have a second call to _pre and _post normalize via the call to html_sanitize :thinking: and still risk to compare apple and orange. no ?

line 1991/2000 
```py
original_value = record[self.name])
get_parsed(html_normalize_post(html_normalize_pre(original_value)) != ...
```